macros/answerHints.pl

```diff
@@ -146,11 +146,16 @@ sub AnswerHints {
       foreach my $wrong (@{$wrongList}) {
 	if (ref($wrong) eq 'CODE') {
 	  if (($ans->{score} < 1 || $options{checkCorrect}) &&
-	      ($ans->{ans_message} eq "" || $options{replaceMessage}) &&
-	      &$wrong($correct,$student,$ans)) {
-	    $ans->{ans_message} = $ans->{error_message} = $message;
-	    $ans->{score} = $options{score} if defined $options{score};
-	    last;
+	      ($ans->{ans_message} eq "" || $options{replaceMessage}) ) {
+	      # Make the call to run the function inside an eval to trap errors
+	      my $myResult = 0;
+	      eval { $myResult = &$wrong($correct,$student,$ans); };
+	      if ( $@ ) { $myResult = 0; }
+	      if ( $myResult ) {
```

Comment:
The space inside the parentheses is inconsistent with the style of the rest of the file.  Those spaces are not preferable in any case according to the coding style that I like!

Reply:
Perhaps perltidy could be run on this file.  I figure eventually we'll run it on all files.

Reply:
Perhaps perltidy could be run on this file.  I figure eventually we'll run it on all files.

Reply:
I agree.  I would like to see that as well.

Reply:
Dropped the space in the second commit.
perltidy applied in the third commit, but I needed to drop `--xci` and `--nlop` from `.perltidy` temporarily, as the version on my Debian machine does not support those options.